mkosi/__init__.py

```diff
@@ -1611,6 +1616,18 @@ def gzip_binary(context: Context) -> str:
     return "pigz" if find_binary("pigz", root=context.config.tools()) else "gzip"
 
 
+def fixup_vmlinuz_location(context: Context) -> None:
+    for d in context.root.glob("boot/vmlinuz-*"):
```

Comment:
Since a later commit does explicitly care for `vmlinux`, this should maybe, too?

Reply:
I think we're good, vmlinux is never shipped in /boot from what I've seen